pages/contributions/contributions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Contributing to the JSON Schema documentation
+section: docs
+---
+
+## Welcome
+
+Welcome to the JSON Schema Contributing Guide, and thank you for your interest. Before working on any contributions, we kindly ask you to [read our Code of Conduct](../contributions/code-of-conduct). 
```

Comment:
I would re-word the later parts like this.

"Welcome to the JSON Schema Contributing Guide, and thank you for your interest. Before working on any contributions, read our Code of Conduct](../contributions/code-of-conduct)."

Reply:
Thanks for the suggestion!